README.md

```diff
@@ -52,7 +52,7 @@ end
 
 #### active_model_serializers
 
-This attribute is an object, all inserted configuration this object it will be passed on to Active Model Serializers, eg.
+This attribute is an object, all inserted configurations of this object it will be passed on to Active Model Serializers, eg.
```

Comment:
@samuelmadeu @SauloSilva 

I think that the correct would be:

> This attribute is an object, all inserted configurations into this object it will be passed on to Active Model Serializers, eg.